backend/wmg/data/extract.py

```diff
@@ -1,11 +1,56 @@
-# TODO generalize
+import os
+
+from backend.corpora.common.corpora_orm import DatasetArtifactFileType
+from backend.corpora.common.entities import Dataset, DatasetAsset
+from backend.corpora.common.utils.db_session import db_session_manager
+
+included_assay_ontologies = {
+    "EFO:0008722": "Drop-seq",
+    "EFO:0010010": "CEL-seq2",
+    "EFO:0010550": "sci-RNA-seq",
+    "EFO:0010961": "Visium Spatial Gene Expression",
+    "EFO:0030002": "microwell-seq",
+    "EFO:0009901": "10x 3' v1",
+    "EFO:0011025": "10x 5' v1",
+    "EFO:0009899": "10x 3' v2",
+    "EFO:0009900": "10x 5' v2",
+    "EFO:0009922": "10x 3' v3",
+    "EFO_0030003": "10x 3' transcription profiling",
+    "EFO:0030004": "10x 5' transcription profiling",
+    "EFO:0008919": "Seq-Well S3",
+}
```

Comment:
We _could_ use `backend.wmg.data.ontology_labels.ontology_term_label()` to obtain the labels and guarantee correctness. If you think that's worthwhile, perhaps just add a TODO for now.

Reply:
I looked at doing that but I'd rather create a function to look up the ontology ids based on the labels. If we have a list of the ids it doesnt actually tell us much at a glance, I figure the dict at least makes it clear what we are filtering on